packages/fast-element/README.md

```diff
@@ -1 +1,5 @@
-# FAST Element
\ No newline at end of file
+# FAST Element
+
+The `fast-element` library is a lightweight solution for easily building performant, memory-efficient, standards-compliant web components. Fast elements work in every major browser and can be used in combination with any front-end framework or even without a framework.
```

Comment:
We capitalize "Web Components" and "Fast Element" in packages/fast-element/docs/building-components.md.